hub/policy/src/main/java/uk/gov/ida/hub/policy/logging/HubEventLogger.java

```diff
@@ -199,7 +205,10 @@ public void logRequestFromHub(SessionId sessionId, String transactionEntityId) {
                 details);
 
         eventSinkProxy.logHubEvent(eventSinkHubEvent);
-        eventEmitter.record(eventSinkHubEvent);
+
+        if (sendToRecordingSystem) {
```

Comment:
Why not have the logic about 'to send or not to send' live in the EventEmitter? That way we reduce the code duplication?

Reply:
Good idea, I will make the changes to Event Emitter.